lib/sandbox/index.js

```diff
@@ -28,5 +28,5 @@ require('./purse');
 
 // Setup the ping-pong and execute routines
 bridge.on('ping', require('./ping').listener('pong'));
-bridge.on('execute', require('./execute').listener('execution.', (typeof console !== 'undefined' ? console : null),
```

Comment:
this change is irrelevant to the PR. it is simply a rename to allow easy coding of non legacy handlers down the line